install.sh

```diff
@@ -22,6 +22,9 @@
 
 set -e
 
+# change directory to the package's root directory
+cd $(dirname $0)
```

Comment:
Please use proper quoting. Both `$0` and the outer expression need to be quoted: `"$(dirname "$0")"`.

Reply:
If --root is given as a relative directory, it will be interpreted according to the directory of install.sh, not $PWD. So we first need to parse the parameters, convert --root to an absolute path, and only then cd.

Reply:
@avikivity Sorry I do not understand why --root is related with this, isn't it --root specifies target directory, not source directory?

Reply:
@avikivity Ah okay I understood, will send new version

Reply:
@syuu1228 I think you forgot to the part about converting `--root` to an absolute address. IOW, you need something like:

```
diff --git a/install.sh b/install.sh
index d2df63a50f..1cd76a88d1 100755
--- a/install.sh
+++ b/install.sh
@@ -51,7 +51,7 @@ upgrade=false
 while [ $# -gt 0 ]; do
     case "$1" in
         "--root")
-            root="$2"
+            root="$(realpath "$2")"
             shift 2
             ;;
         "--prefix")
```

And you have to do that *before* changing the current directory.